controllers/dockermachine_controller.go

```diff
@@ -115,6 +115,12 @@ func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		return ctrl.Result{}, err
 	}
 
+	//delete dockerMachine
+	if !machine.ObjectMeta.DeletionTimestamp.IsZero() {
+		log.Info("Start delete dockerMachine")
+		return r.delete(ctx, cluster, machine, dockerMachine)
+	}
+
 	// creating loadbalancer
 	log.Info("Reconciling cluster")
 	elb, err := getExternalLoadBalancerNode(cluster.Name, log)
```

Comment:
@chuckha Also unrelated to this PR, but shouldn't the LB creation be part of the DockerCluster controller rather than DockerMachine?

Reply:
This will be fixed in https://github.com/kubernetes-sigs/cluster-api-provider-docker/issues/152